### PR TITLE
APM > .NET > Update to version 2.0

### PR DESCRIPTION
**Pull request by andrewlock**
### What does this PR do?
Document the latest features and configuration details for .NET tracer

### Motivation
There were a number of breaking changes in version 2.0 of the .NET tracer; we want to make sure we have the correct documentation for the current version of the tracer.

### Preview
https://docs-staging.datadoghq.com/andrewlock/version-2-updates/tracing/setup_overview/compatibility_requirements/dotnet-core
https://docs-staging.datadoghq.com/andrewlock/version-2-updates/tracing/setup_overview/compatibility_requirements/dotnet-framework/
https://docs-staging.datadoghq.com/andrewlock/version-2-updates/tracing/setup_overview/setup/dotnet-core
https://docs-staging.datadoghq.com/andrewlock/version-2-updates/tracing/setup_overview/setup/dotnet-framework/

### Additional Notes
Version 2 has not been finished yet, so we may make additional changes before merging

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

@DataDog/apm-dotnet

### CODE REVIEW

File: content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md [hunk 1]
```diff
@@ -50,24 +51,25 @@ The [latest version of the .NET Tracer][4] can automatically instrument the foll
...
 | RabbitMQ                        | `RabbitMQ.Client` 3.6.9+                                                                  | `RabbitMQ`           |
 | Redis (ServiceStack client)     | `ServiceStack.Redis` 4.0.48+                                                              | `ServiceStackRedis`  |
 | Redis (StackExchange client)    | `StackExchange.Redis` 1.0.187+                                                            | `StackExchangeRedis` |
 | Service Fabric Remoting         | `Microsoft.ServiceFabric.Services.Remoting` 4.0.470+                                      | `ServiceRemoting`    |
-| SQL Server                      | `System.Data` 4.0.0+</br>`System.Data.SqlClient` 4.0.0+</br>`Microsoft.Data.SqlClient` 1.0.0+  | `AdoNet`             |
+| SQLite                          | `System.Data.Sqlite` 2.0.0+ </br>`Microsoft.Data.Sqlite` 1.0.0+                           | `Sqlite`             |
+| SQL Server                      | `System.Data` 4.0.0+</br>`System.Data.SqlClient` 4.0.0+</br>`Microsoft.Data.SqlClient` 1.0.0+  | `SqlClient`     |
```
Comment (lucaspimentel) [inline content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md:72]: [comment for line 80 below, github won't let me comment directly]
```diff
- We recommend using the latest patch version of .NET Core 3.1 or .NET 5.
+ Datadog recommends using the latest patch version of .NET Core 3.1, .NET 5, or .NET 6.
```

File: content/en/tracing/setup_overview/setup/dotnet-core.md [hunk 2]
```diff
@@ -467,14 +455,23 @@ Enables or disables a specific integration. Valid values are: `true` or `false`.
...
 
+#### Deprecated settings
+
+`DD_TRACE_LOG_PATH`
+: Sets the path for the automatic instrumentation log file and determines the directory of all other .NET Tracer log files. Ignored if `DD_TRACE_LOG_DIRECTORY` is set. 
+
+`DD_TRACE_ADONET_EXCLUDED_TYPES`
+: **TracerSettings property**: `AdoNetExcludedTypes` <br>
+Sets a list of `AdoNet` types (for example, `System.Data.SqlClient.SqlCommand`) that will be excluded from automatic instrumentation. Unused in version 2.0+.
+
```
Comment (lucaspimentel) [inline content/en/tracing/setup_overview/setup/dotnet-core.md:470]: This setting was removed, not deprecated.
```suggestion
```

File: content/en/tracing/setup_overview/setup/dotnet-framework.md [hunk 3]
```diff
@@ -378,10 +374,7 @@ Added in version 1.17.0.
 
 `DD_TRACE_LOG_DIRECTORY`
 : Sets the directory for .NET Tracer logs. <br>
-**Default**: `%ProgramData%\Datadog .NET Tracer\logs\`
-
-`DD_TRACE_LOG_PATH`
-: Sets the path for the automatic instrumentation log file and determines the directory of all other .NET Tracer log files. Ignored if `DD_TRACE_LOG_DIRECTORY` is set.
+**Default**: `%ProgramData%\Datadog .NET Tracer\logs\` on Windows, `/var/log/datadog/dotnet` on Linux
```
Comment (lucaspimentel) [inline content/en/tracing/setup_overview/setup/dotnet-framework.md:377]: We can revert this change. .NET Framework does not run on Linux.
```suggestion
**Default**: `%ProgramData%\Datadog .NET Tracer\logs\`
```

**Review by lucaspimentel — COMMENTED** [review]
👍🏽

**Review by sarina-dd — APPROVED** [review]
